app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1045,12 +1048,7 @@ private void startFormEntry(AndroidSessionWrapper state) throws SessionUnavailab
                 platform = CommCareApplication._().getCommCarePlatform();
             }
 
-            //TODO: May need to pass session over manually
             formEntry(platform.getFormContentUri(record.getFormNamespace()), record, CommCareActivity.getTitle(this, null));
-
-        } catch (StorageFullException e) {
-            throw new RuntimeException(e);
-        }
```

Comment:
Removed this try/catch because StorageFullException extends RuntimeException, so why catch and re-raise?
I can add it back in if you see importance to this.